youtube_dl/extractor/canvas.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+
+
+class CanvasIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)?(?:www\.)?canvas\.be/video/(?P<id>.+)'
+    _TEST = {
+        'url': 'http://www.canvas.be/video/de-afspraak/najaar-2015/de-afspraak-veilt-voor-de-warmste-week',
+        'md5': 'ea838375a547ac787d4064d8c7860a6c',
+        'info_dict': {
+            'id': 'de-afspraak/najaar-2015/de-afspraak-veilt-voor-de-warmste-week',
+            'title': 'De afspraak veilt voor de Warmste Week',
+            'ext': 'mp4',
+            'duration': 49,
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+
+        title = self._search_regex(
+            r'<h1 class="video__body__header__title">(.*?)</h1>', webpage,
+            'title')
+        data_video = self._html_search_regex(
+            r'data-video="(.+)"', webpage, 'data-video')
```

Comment:
@dstftw This gives me another `DownloadError: Unable to extract data-video;`. I actually don't understand why it should be this (`r'data-video=(["\'])(?P<id>.+?)\1\''`) regex instead of `r'data-video="(.+)"'` because the latter will find the complete `data-video` value which I can not see how the former can do that.